### PR TITLE
fix: remove old links and redirects

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,11 +307,10 @@ Services to securely store your Docker images.
...
-* [Collecting docker logs and stats with Splunk](https://www.splunk.com/blog/2015/08/24/collecting-docker-logs-and-stats-with-splunk/)
+* [Collecting docker logs and stats with Splunk](https://www.splunk.com/blog/2015/08/24/collecting-docker-logs-and-stats-with-splunk.html)
 * [CoScale](https://www.coscale.com/docker-monitoring) - Full stack monitoring for containerized applications and microservices. Powered by anomaly detection to find performance problems faster.
 * [Datadog](https://www.datadoghq.com/) - Datadog is a full-stack monitoring service for large-scale cloud environments that aggregates metrics/events from servers, databases, and applications. It includes support for Docker, Kubernetes, and Mesos.
 * [Meros](https://meros.io) - Analyzes containers resources, captures logs, remote web SSH terminal and powerful DevOps alerts.
-* [New Relic](https://newrelic.com/partner/docker) - New Relics Docker Monitoring tool
```
Comment (agebhar1) [inline README.md]: I also had a look yesterday for a replacement but didn't find any - are you aware of one?

File: README.md [hunk 2]
```diff
@@ -453,7 +452,6 @@ Services to securely store your Docker images.
 * [Dockly](https://github.com/swipely/dockly) - Dockly is a gem made to ease the pain of packaging an application in Docker by [@swipely](https://github.com/swipely/)
 * [dockramp](https://github.com/jlhawn/dockramp) - Proof of Concept: A Client Driven Docker Image Builder by [@jlhawn](https://github.com/jlhawn)
 * [draw-compose](https://github.com/Alexis-benoist/draw-compose) - Utility to draw a schema of a docker compose by [@Alexis-benoist](https://github.com/Alexis-benoist)
-* [Dropdock](http://dropdock.io/) - A framework designed for Drupal to build fast, isolated development environments using Docker.
```
Comment (agebhar1) [inline README.md]: The website is down and the last commit on the corresponding GitHub project https://github.com/twinbit/dropdock is 2015/08 - might be a good reason to drop the link.

File: README.md (hunk 1 above)
Comment (veggiemonk) [inline README.md]: new relic still offers docker monitoring but it is under the linux server monitoring.

Comment (Moshe-Immerman) [inline README.md]: Doesn't seem like they do an awesome job of it, or even promote it much (I can't even find a heading to link)- From what I can see they just add a few tags onto the processes running inside the containers and then you need todo custom grouping on process data.

Comment (veggiemonk) [inline README.md]: yes you are right, let's remove it then!

**Pull request merged**